tests/unit/statistical_testing/internals/CrossValidationMMD_unittest.cc

```diff
@@ -63,8 +64,14 @@ TEST(CrossValidationMMD, biased_full)
 	const float64_t alpha=0.05;
 	const auto stype=ST_BIASED_FULL;
 
+	std::mt19937_64 prng(seed);
+
 	auto gen_p=some<CMeanShiftDataGenerator>(0, dim, 0);
 	auto gen_q=some<CMeanShiftDataGenerator>(difference, dim, 0);
+	// FIXME: Random Refactor PR
+	// FIXME: CMeanShiftDataGenerator is of Feature type, which is immutable
+	gen_p->CSGObject::put("seed", seed);
+	gen_q->CSGObject::put("seed", seed);
```

Comment:
CMeanShiftDataGenerator uses the RandomMixin, but it's immutable (derived from CFeatures), so can't use it's version of "put"

Reply:
can you explain this to me? :)